andela/pippin-ah-backend
# [Feature: #162727575] Track Comment Edit History

#### What does this PR do?
This PR adds a feature for editing, deleting, getting and tracking comments edit history.

#### Description of Task to be completed?
A user should be able to see comments and the previous versions of comments made on an article after previous edits.

#### How should this be manually tested?
- run `db:migrate`
- send a `PATCH` request to `/api/v1/articles/:slug/comments/:id`
- the comment should be successfully updated

- send a `GET` request to `/api/v1/articles/:slug/comments/:id/edits`
- you should get a response including the comment edit history

- after creating a comment, send a `DELETE` request to `/api/v1/articles/:slug/comments/:id`
- the comment should be deleted successfully

#### What are the relevant pivotal tracker stories?
[#162727575](https://www.pivotaltracker.com/story/show/162727575)
